src/Root.sol

```diff
@@ -30,6 +34,7 @@ contract Root is Auth {
     event CancelRely(address indexed target);
     event RelyContract(address indexed target, address indexed user);
     event DenyContract(address indexed target, address indexed user);
+    event RecoverTokens(address indexed fromContract, address indexed toUser, address indexed token, uint256 amount);
```

Comment:
Compared to other messages, plus using the transfer method ordering, I would name it like this now:
```suggestion
    event RecoverTokens(address indexed target, address indexed token, address indexed to, uint256 amount);
```